webpack.config.js

```diff
@@ -4,6 +4,7 @@ const SRC_DIR = path.join(__dirname, './client/src');
 var DIST_DIR = path.join(__dirname, './client/dist');
 
 module.exports = {
+  devtool: 'source-map',
```

Comment:
nice - source maps are always useful. Not sure how they really debugged their front end before

Reply:
Just found out about it like last week and it's super dope. You can put breakpoints in React whattttttt